PyQt5-stubs/QtWidgets.pyi

```diff
@@ -3828,7 +3828,7 @@ class QGraphicsDropShadowEffect(QGraphicsEffect):
     colorChanged: QtCore.pyqtSignal
     blurRadiusChanged: QtCore.pyqtSignal
     offsetChanged: QtCore.pyqtSignal
-    def setColor(self, color: typing.Union[QtGui.QColor, QtCore.Qt.GlobalColor, QtGui.QGradient]) -> None: ...
+    def setColor(self, color: typing.Union[QtGui.QColor, QtCore.Qt.GlobalColor]) -> None: ...
```

Comment:
Seems maybe odd but this isn't anything we customized seemingly.

Reply:
For whatever reason, a lot of `QColor`-taking functions are annotated as also accepting `QGradient`s, but don't work

```pycon
effect = QGraphicsDropShadowEffect()
gradient = QGradient()
effect.setColor(gradient)
```
```pytb
Traceback (most recent call last):
  File "<input>", line 1, in <module>
TypeError: setColor(self, Union[QColor, Qt.GlobalColor, QGradient]): argument 1 has unexpected type 'QGradient'
```

Reply:
I guess this is them fixing one that we hadn't.

Reply:
https://github.com/stlehmann/PyQt5-stubs/pull/29/files#diff-92315caecc7de3d85e401ce18dec0ac4R3798

Strange. I had already fixed this specific one. My PR was merged, but somehow that line didn't make it into master 🤔:

https://github.com/BryceBeagle/PyQt5-stubs/blob/c17297d85e7922bafe0965970d12a6966a19d575/PyQt5-stubs/QtWidgets.pyi#L3900